pallets/gear/src/manager/result.rs

```diff
@@ -0,0 +1,50 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2022 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use sp_runtime::{DispatchError, ModuleError};
+
+/// Handle the errors from the `GasTree` operations
+///
+/// # NOTE
+///
+/// Only handles `NodeAlreadyExists` for now.
+pub fn gas_tree(e: DispatchError) {
```

Comment:
all this stuff seems excess for me.
all gas tree operations should be unreachable if they correctly used